pvm.py

```diff
@@ -9,14 +9,23 @@
 from logging.handlers import TimedRotatingFileHandler
 import sys
 
-# Place your videos in this folder for autostart
+# Initialize global variables
 HOME = str(Path.home()) + "/"  # The home directory, e.g. /home/pi/
 LOG_PATH = HOME + "PVM/log/"
-PREFIX_PATH = HOME + "Videos/"
-VIDEO_PATH = "jellyfish720p.mp4"
+PEFIX_PATH = HOME + "Videos/" # Place your videos in this folder for autostart
+VIDEO_PATH_ONE = "jellyfish720p.mp4"
+VIDEO_PATH_TWO = "jellyfish720p.mp4"
```

Comment:
```suggestion
VIDEO_PATH_ONE = ""
VIDEO_PATH_TWO = ""
```

no need to keep `jellyfish720p.mp4` around